gnssanalysis/gn_io/sp3.py

```diff
@@ -579,13 +579,16 @@ def sp3merge(
     :return pd.DataFrame: The merged sp3 DataFrame.
     """
     sp3_dfs = [read_sp3(sp3_file, nodata_to_nan=nodata_to_nan) for sp3_file in sp3paths]
+    # Create a new attrs dictionary to be used for the output DataFrame
+    merged_attrs = merge_attrs(sp3_dfs)
```

Comment:
It looks like to me that the merge_attrs is quite simplistic. 
if values in the header are different, replace most of the values by "MIX". 
Maybe a better strategy, 
start date = min(start dates)
n_epochs = sum(n_epochs)
....

Reply:
Yeah I didn't look too much into that `merge_attrs` function - seems like it was an MVP then.
Not sure for some of these header fields, but `DATETIME` and `N_EPOCHS` I can definitely be write out by analysing the data of the incoming DataFrames.

An example block is given below:
```
VERSION                                 c
PV_FLAG                                 P
DATETIME     2023 10 29  0  0  0.00000000
N_EPOCHS                              288
DATA_USED                             u+U
COORD_SYS                           IGS20
ORB_TYPE                              FIT
AC                                    GFZ
FILE_TYPE                               M
TIME_SYS                              GPS
```
Not sure if there are any other fields where `MIX` is not appropriate?

Reply:
I could perhaps label some fields with just a new string that incorporates the ones from the original files
- e.g. for `VERSION` we could label as `c+d` when the two files are Version `c` and `d`
- or the AC could be `GFZ+WHU` when combining those two
It won't tell us which part of the data comes from each, but at least gives the user an idea of what is in it

Reply:
Ok, just pushed through a commit that allows the DATETIME and N_EPOCHS fields to be filled properly - based on the data in the new merged file. 
For some of the other fields, I've had to use a certain number of characters, so I couldn't achieve my aim of just appending each instance with a `+`, although I've tried to get as close to that aim as possible (e.g. `AC` uses all 4 chars available and combines 2 chars from each AC)

Also, the way this function originally outputted the first line, it couldn't be re-read by our `read_sp3` function. I think I've got it working now so that we can read the outputs again